karma-build.conf.js

```diff
@@ -9,6 +9,7 @@
 module.exports = function (config) {
   require('./karma-base.conf.js')(config);
   config.files.push('build/test/wtf_mock.js');
+  config.files.push('build/test/custom_error.js');
```

Comment:
I add this file just to simulate a Error with customized method before patched by zone.js. so I can test the customized method is still valid after zone.js patch Error with ZoneAwareError.